app/src/controllers/project/sagas.js

```diff
@@ -0,0 +1,75 @@
+import { takeEvery, all, put, select, call } from 'redux-saga/effects';
+import { URLS } from 'common/urls';
+import { showNotification, NOTIFICATION_TYPES } from 'controllers/notification';
+import { activeProjectSelector } from 'controllers/user';
+import { fetch } from 'common/utils';
+
+import { UPDATE_DEFECT_SUBTYPE, ADD_DEFECT_SUBTYPE } from './constants';
+import { updateDefectSubTypeSuccessAction, addDefectSubTypeSuccessAction } from './actionCreators';
+
+function* updateDefectSubType({ payload: subType }) {
+  try {
+    const activeProject = yield select(activeProjectSelector);
```

Comment:
Please use `projectIdSelector` from `controllers/pages` to successfully send requests from project settings page on Admin part.

Reply:
done